### PR TITLE
Milestone 3 styling

### PR DESCRIPTION
**Pull request by lekegitrepo**
##### Set default width and color for the button component.

### CODE REVIEW

File: src/components/button-panel.js [hunk 1]
```diff
@@ -4,34 +4,34 @@ import Button from './button';
...
-      <Button name='0' className='col-md-6' />
-      <Button name='.' className='col-md-3' />
-      <Button name='=' className='col-md-3' />
+    <div className='row btn-group'>
+      <Button name='0' btn='btn' wide color='grey' />
+      <Button name='.' btn='btn' color='grey' />
+      <Button name='=' btn='btn' />
     </div>
   </div>
 );
```
Comment (jofer86) [inline src/components/button-panel.js:37]: - Let's try to make the rendering of your button panel more dynamic, you can render the entire component with the button components with a loop taking the info from an array or an object, that would save up several lines of code.

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello!, I'm Jorge, and  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- Your display component's height is actually 120px and needs to be 100px 
image
image
- Other than that you are good to go, just a minor style suggestion, please do check the comments in your code.
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

File: src/components/button-panel.js (hunk 1 above)
Comment (lekegitrepo) [reply src/components/button-panel.js:37]: **_I'll refactor and add the dynamic implementation of the button panel later. However, thanks for the suggestion @jofer86._**

**Review by shloch — APPROVED** [review]
## Hello @lekegitrepo 
Good to go............

accomplished

On to the next milestone

**Pull request merged**